lib/utils/email.js

```diff
@@ -9,22 +9,22 @@ const transporter = nodemailer.createTransport({
     pass: process.env.NOTBOTPASSWORD, // NotBot Admin gmail password
   },
 });
-
+const sendInvite = email => {
 // invite email 
-const invite = {
-  from: '"NotBot Admin" <behumannotbot@gmail.com>', // sender address
-  to: 'bryanakitchen@gmail.com, francopdx@gmail.com, jillian.l.gibson@gmail.com, shaneupchurch@gmail.com, fernandclay@gmail.com',  // list of receivers
-  cc: 'behumannotbot@gmail.com', 
-  subject: 'test email', // Subject line
-  text: 'this is a test of the admin invite link.', // plain text body
-  html: '<a href="http://api/v1/auth/invite">Invite Link</a>', // html body
+  const invite = {
+    from: '"NotBot Admin" <behumannotbot@gmail.com>', // sender address
+    to: email,  // list of receivers
+    cc: 'behumannotbot@gmail.com', 
+    subject: 'beHuman Admin Invite', // Subject line
+    text: 'admin invite link.', // plain text body
+    html: 'Use this <a href="https://be-human-demo-staging.herokuapp.com">Invite Link</a> to access the admin page. You will need to use the password: adminPassword and this email to login.', // html body
```

Comment:
So can anyone go to this link (our heroku link) and type in adminPassword and login? Or will this only work for our specific email addresses?

Reply:
Won't work for just anyone - I can explain why during our stand up today. :)